$NetBSD: patch-CMakeLists.txt,v 1.2 2019/10/03 15:53:21 nia Exp $

Hunk 1:

Use /lib for "private libs" (liblightspark). This may be evil,
but currently the private rpath isn't being added properly.

Hunk 2:

Install config file as example.

--- CMakeLists.txt.orig	2019-09-15 15:33:56.000000000 +0000
+++ CMakeLists.txt
@@ -173,10 +173,10 @@ IF(UNIX)
    SET(DATADIR "share")
    # Absolute path, because this path is embedded in the binary
    SET(LSDATADIR "${CMAKE_INSTALL_PREFIX}/${DATADIR}/lightspark")
-   SET(LIBDIR "lib${LIB_SUFFIX}")
+   SET(LIBDIR "lib")
    #We need a absolute path here because we use it for rpath
    #later on
-   SET(PRIVATELIBDIR "${CMAKE_INSTALL_PREFIX}/${LIBDIR}/lightspark")
+   SET(PRIVATELIBDIR "${CMAKE_INSTALL_PREFIX}/${LIBDIR}")
    SET(CMAKE_INSTALL_RPATH "${PRIVATELIBDIR}")
 ELSE()
   SET(ETCDIR ".")
@@ -560,7 +560,7 @@ GETTEXT_CREATE_TRANSLATIONS(i18n/lightsp
 
 if(UNIX)
 	INSTALL(FILES etc/xdg/lightspark.conf
-		      DESTINATION ${ETCDIR}/xdg/)
+		      DESTINATION ${CMAKE_INSTALL_PREFIX}/share/examples/lightspark/xdg)
 ELSE()
   INSTALL(FILES COPYING DESTINATION "." RENAME COPYING.txt)
   INSTALL(FILES COPYING.LESSER DESTINATION "." RENAME COPYING.LESSER.txt)
